libs/remix-ui/terminal/src/lib/components/RenderCall.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react'  // eslint-disable-line
+
+import helper from 'apps/remix-ide/src/lib/helper'
+import CheckTxStatus from './ChechTxStatus' // eslint-disable-line
+import showTable from './Table'
+import { ModalDialog } from '@remix-ui/modal-dialog' // eslint-disable-line
+
+const remixLib = require('@remix-project/remix-lib')
+var typeConversion = remixLib.execution.typeConversion
+
+const RenderCall = ({ tx, resolvedData, logs, index, plugin, showTableHash, txDetails }) => {
+  const to = resolvedData.contractName + '.' + resolvedData.fn
+  const from = tx.from ? tx.from : ' - '
+  const input = tx.input ? helper.shortenHexData(tx.input) : ''
+  const txType = 'call'
+
+  const debug = (event, tx) => {
+    event.stopPropagation()
+    if (tx.isCall && tx.envMode !== 'vm') {
+      return (<ModalDialog
```

Comment:
Instead of returning modal, set state to show modal. That way we don't load multiple modals to the DOM.